### PR TITLE
Revises the landing page. Shifts some pages around again. [Fixes #255]

### LINKED ISSUE #255: Landing page revision
Given that OPM may actually link to smeqa.usds.gov (miracles do happen!) we want to revise the landing page with the following changes:

* [ ] Remove the large photo and move it to the /about/our-process/ page
* [x] Rename /about/our-process to /about/smeqa-principles/
* [x] Take the top half of the /hiring-phases/ overview page and move it to the top of the landing page
* [x] Return /hiring-phases/getting-started to be the index page for /hiring-phases/
* [x] Under the "The process" graphic split the two paragraphs that are on currently on the landing page into two columns
* [x] Add links to the MW memo and EO under those two paragraphs
* [x] Add "Start now" and "Learn more" CTAs 
* [x] Add a "Is SME-QA right for you?" or something like that as a new section with 3 cards under it that speak to evaluation:
  * About 
  * Why it's legal
  * Results

### PR DESCRIPTION
**Pull request by humancompanion-usds**

### CODE REVIEW

File: _data/navigation.yml [hunk 1]
```diff
@@ -10,16 +10,14 @@ primary:
     name: results
 
 about:
-  - text: The SME-QA Process
-    href: /about/our-process/
+  - text: SME-QA Pinciples
```
Comment (wslack) [inline _data/navigation.yml:13]:
```suggestion
  - text: SME-QA Principles
```

**Comment by wslack** [comment]
Ooooooh.

**Comment by wslack** [comment]
Seems like the two "read more" links should be bullets and after the two larger paragraphs - right now, my eye isn't clear which direction to follow the text.

**Comment by wslack** [comment]
If Stephanie agrees I can do some of the adjustments ^^ (but don't need to)

**Comment by humancompanion-usds** [comment]
Have another look @wslack and let me know if that addresses the issues.

**Comment by wslack** [comment]
@humancompanion-usds this is wonderful! We have some text edits to make later; the only edit for you is adding a caption under the image to make it clearer (in larger text than on the image) that the given numbers are examples.

Otherwise our eye is just drawn to the footer and the work in https://github.com/usds/SME-QA-Website/issues/257 - we will update the text.

**Comment by humancompanion-usds** [comment]
@wslack - I updated the images to bump up all of the type sizes which were too small for this resolution. May mess with them a bit more, but these should work. I'll tackle the footer next but can we get the text changes in today or do those later? I'd like to push this out before I move onto other things.

**Pull request merged**